lib/rest-connector.js

```diff
@@ -100,31 +102,12 @@ function RestConnector(baseURL, settings) {
   this._settings = settings || {};
 
   // Extract the default options for request module
-  var options = this._settings.options ||
-    this._settings.defaults || this._settings;
+  var options = omit(defaults(this._settings.options || {},
+                              this._settings,
+                              this._settings.defaults),
+                     ['defaults', 'connector', 'operations']);
 
-  var defaults = {};
-
-  if (options) {
-    if (options.clientCert) {
-      defaults.cert = options.clientCert;
-    }
-    if (options.clientKey) {
-      defaults.key = options.clientKey;
-    }
-
```

Comment:
Good catch.  Added in the mapping to ensure keys are included and a test for it.